### PR TITLE
[RFR] Added “Flux” entry

### PR DESCRIPTION
**Pull request by CrocoDillon**
I tried to keep it minimal as any further explaining would only confuse any reader with no experience with Flux... unless we would elaborate but then it would not be simplified anymore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,10 @@ The versioning convention of ECMAScript has been the subject of hot debates. We
 
 [Express](http://expressjs.com/en/index.html) is a fast, unopinionated, minimalist web framework for [Node.js](#nodejs). Express provides a thin layer of fundamental web application features, without obscuring Node.js features that developers already know and like. The myriad of HTTP utility methods and middleware provided by Express makes creating a robust API quick and easy.
 
+## Flux
+
+[Flux](https://facebook.github.io/flux/) is an application structure that is developed and used at Facebook to complement Reacts one-way data flow. With Flux application state and logic are contained in stores.
```
Comment (KittyGiraudel) [inline README.md]: Making _React_ a link to `#react` would be nice. :)
Also _React’s_, not _Reacts_.
Missing comma after _With Flux_.

Comment (CrocoDillon) [inline README.md]: All fixed!

**Comment by KittyGiraudel** [comment]
Thanks! :)

**Comment by CrocoDillon** [comment]
Any time! :smile:

**Pull request merged**